src/sweeper/sweepers/metadata.py

```diff
@@ -17,7 +17,7 @@
 
 #: these constants were copied from https://github.com/agrc/agol-validator/blob/master/validate.py
```

Comment:
**question**: is this still true? Are they still synchronized?

Reply:
They are now with the exception of `ugrc`. Auditor is missing that tag in main but it looks like [Jake is on top of it](https://github.com/agrc/auditor/pull/62/files#diff-77cebe02e1f55354178f02ed887d6123ae7e30527a2a243c17ae2dd9c1645021).